### ISSUE TITLE
GPU-Z returns Unknown version

### ISSUE THREAD

**Issue by Zazou89** [microsoft/winget-pkgs#22086]
Create issues related to the winget.exe client [here](https://github.com/microsoft/winget-cli/issues/new?template=Bug_Report.md)

# Brief description of your issue
GPU-Z returns an unknown version, last version in installed (2.40.0)
# Steps to reproduce

winget install gpu-z

# Expected behavior

Display the right version instead of Unknwon. 

# Actual behavior

![image](https://user-images.githubusercontent.com/206181/126678606-b3e3c32b-daa8-4ab7-8002-4c34852f37cb.png)

# Environment

```none
[winget --info]
Windows Package Manager version : v1.0.11694
Windows: Windows.Desktop version : v10.0.22000.71
Package: Microsoft.DesktopAppInstaller version : v1.15.11694.0

Any other software?
```

**Comment by ImJoakim** [comment]
This often happens when the installer doesn't assert a version to Windows in "Add / Remove Programs".

![image](https://user-images.githubusercontent.com/52826317/126679082-3a472c02-6a00-48f9-9acf-35736137e43b.png)

**Comment by ImJoakim** [comment]
This reply might be useful in regards to how this can be resolved: https://github.com/microsoft/winget-pkgs/issues/16003#issuecomment-872653235

**Comment by Zazou89** [comment]
Thank you @ImJoakim, i'll try to contact the developer ;)

**Comment by W1zzardTPU** [comment]
GPU-Z author here, thanks for letting me know, I'll add the version # in future releases

**Comment by W1zzardTPU** [comment]
https://www.techpowerup.com/forums/threads/gpu-z-test-build-fixed-installer-for-winget.284822/

Please confirm that these are the required changes for winget

**Comment by ImJoakim** [comment]
> https://www.techpowerup.com/forums/threads/gpu-z-test-build-fixed-installer-for-winget.284822/
> 
> Please confirm that these are the required changes for winget

@W1zzardTPU Yep! That should do it :)

![image](https://user-images.githubusercontent.com/52826317/126701719-15bf4c2e-9dc1-40a9-8a74-17fec3a7d7dc.png)

**Comment by walderston** [comment]
This is fixed in #24084 
![image](https://user-images.githubusercontent.com/59405120/129271049-9db666c4-a6b3-474e-a1e1-366f86c85361.png)

Believe this can be closed

**Comment by denelon** [comment]
Thanks @W1zzardTPU
